website/docs/d/catalog.html.markdown

```diff
@@ -37,8 +37,25 @@ resource "vcd_catalog_item" "myItem" {
 The following arguments are supported:
 
 * `org` - (Optional, but required if not set at provider level) Org name 
-* `name` - (Required) Catalog name
+* `name` - (Required) Catalog name. (Optional when `filter` is used)
```

Comment:
I think that for someone scanning, the current layout is better, as the field descriptions are all visible after a single word between parentheses. Using your suggestion we would break the symmetry and make it more difficult to scan. But I am open to different opinions.